src/readCSV.c

```diff
@@ -80,7 +78,10 @@ struct data readCSV(const char * filename, int skipLines) {
   memset(value,0,sizeof(double)*arraySize);
 
   for (i=0; i<skipLines; i++) {
-    fgets(buf,100,fp); // skip the first "skipLines" lines
+    // skip the first "skipLines" lines
+    if (fgets(buf,100,fp) == NULL){
+      exit(1);
```

Comment:
How is the error that leads to a non-zero exit code reported to the user? It looks like line 82 is true if the EOF is encountered *or* an error is encountered. This must be reported to stderr, no? If not, how would a user understand why the program exits with an error?